packages/core/src/interactions/packet/acknowledgement.ts

```diff
@@ -117,7 +117,7 @@ export function subscribeToAcknowledgements(
   protocolAck: string
 ) {
   const limitConcurrency = oneAtATime<void>()
-  subscribe(
+  await subscribe(
```

Comment:
Is this an `await` on a non-promise?

Reply:
`libp2p.handle()` became asynchronous, so `subscribe` becomes asynchronous as well

Reply:
Correct.